### PR TITLE
Release version 1.1.3 of Orsetto.

### PR DESCRIPTION
**Pull request by jhwoodyatt**
Release Conjury 1.1.3.

### CODE REVIEW

**Comment by jhwoodyatt** [comment]
Hi, so I have a problem. This version of Orsetto requires a version of the Conjury OMake library that cannot build on systems without OCaml native mode compilers. That's why it currently fails on some of the supported architectures for OCaml 5.0. If the policy is not to merge releases that cannot build on `bytecode-only` switches, then I need figure out how to make this package unavailable on those switches. Alas, I've studied the OPAM documentation, and it doesn't seem possible to make a package that excludes `bytecode-only` compilers.

Please advise?

**Comment by jhwoodyatt** [comment]
Hi OPAM Repository Curators,

I have a problem attempting to release Orsetto 1.1.3. This release is a patch update to correct a couple of errors that were preventing the 1.1 series from compiling on OCaml 5.0, but it still uses the older Conjury V2 build logic that doesn't have support for `bytecode-only` switches. Accordingly, the OPAM-CI check is reporting failures attempting to build Orsetto 1.1.3 on the various architectures and platforms that OCaml 5.0 currently supports only with `bytecode-only` switches.

It will be a few weeks more before I can release Orsetto 1.2, which uses the newer Conjury V3 build logic that supports `bytecode-only` switches, and which also contains a few additional feature improvements. My recommendation is that you allow Orsetto 1.1.3 to be released despite it still being broken on `bytecode-only` switches. (It has never worked on `bytecode-only` switches.)

**Comment by mseri** [comment]
That's not a problem. Can you add
```
conflicts: [ "ocaml-option-bytecode-only" ]
```
in the affected opam files?

File: packages/orsetto/orsetto.1.1.3/opam [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+]
+license: "BSD-2-Clause"
+depends: [
+    "ocaml" { >= "4.08.1" }
+    "conjury" { build & >= "2.2" & < "3.0~" }
+    "omake" { build & >= "0.10.3" }
+    "uucd" { build & = "15.0.0" }
+    "ounit2" { build & with-test & >= "2.2" }
+]
+build: [
```
Comment (mseri) [inline packages/orsetto/orsetto.1.1.3/opam:46]:
```suggestion
conflicts: [ "ocaml-option-bytecode-only" ]
build: [
```

**Comment by jhwoodyatt** [comment]
Test failures on those `macOS` Homebrew "experimental" targets appear to be related to OPAM-CI issues (the test user does not appear to have permission to access Homebrew API cache). It doesn't seem to me that the problem is with the Orsetto package on those targets. Do you agree?

**Comment by mseri** [comment]
That's fine, thanks a lot

**Pull request merged**